tests/test-helpers/src/builder.rs

```diff
@@ -231,14 +231,16 @@ impl TestSetupBuilder {
             ..StateConfig::default()
         };
 
-        let config = Config {
+        let mut config = Config {
             utxo_validation: self.utxo_validation,
             txpool: fuel_core_txpool::config::Config::default(),
             block_production: self.trigger,
             starting_gas_price: self.starting_gas_price,
             ..Config::local_node_with_configs(chain_conf, state)
         };
-        assert_eq!(config.combined_db_config.database_type, DbType::RocksDb);
+        config.combined_db_config.max_database_cache_size = Some(16 * 1024 * 1024 * 1024);
+        config.combined_db_config.max_fds = -1;
+        config.combined_db_config.database_type = DbType::RocksDb;
```

Comment:
It was a fast way how to enable in benchmarks caching, but in this PR we need to enable it from the benchmark side, not for all tests=) Plus, in the case of benchmark we want to start use `OnCreation` policy

Reply:
In benchmarks we will use the `OnCreation` policy thanks to this change : https://github.com/FuelLabs/fuel-core/pull/2526/files#diff-a287ac1ddb3103008d127dec29efad3be71aad1dc35692ea2b110739027f2a74R66

Reply:
The rest of the config has been moved to benches